test/fixtures/cookbooks/jenkins_user/recipes/delete.rb

```diff
@@ -1,11 +1,8 @@
 include_recipe 'jenkins_server_wrapper::default'
 
-# Create a simple user
-jenkins_user 'sethvargo'
-
-# Delete an existing user
+# Create and Delete an existing user
 jenkins_user 'sethvargo' do
-  action :delete
+  action [:create, :delete]
```

Comment:
You probably wanna use the name property in the resource to made the existing resources uniquely named while they still both create/delete vargo. That way you still get 2 unique resources firing here.